File: problems/62/Main.py
import math

#--SPLIT--
#-class Solution:
    # @param a : integer
    # @param b : integer
    # @return an integer
#-    def uniquePaths(self, a, b):



#--SPLIT--
class Solution:
    # @param a : integer
    # @param b : integer
    # @return an integer
    def uniquePaths(self, a, b):
        return 0


#--SPLIT--
if __name__ == "__main__":
    test_cases = int(raw_input())
    for i in range(test_cases):
        lineItems = [int(l) for l in raw_input().split(" ")]
        a, b = lineItems
        s = Solution()
        print(s.uniquePaths(a, b))

File: problems/62/Main.cpp
#include <iostream>
#include <cstdio>
#include <algorithm>
#include <cstring>
#include <string>
#include <cctype>
#include <stack>
#include <queue>
#include <list>
#include <vector>
#include <map>
#include <sstream>
#include <cmath>
#include <bitset>
#include <utility>
#include <set>
#include <numeric>
#include <ctime>
#include <math.h>

using namespace std;

class Solution {
	public:
		int uniquePaths(int a, int b);
};

//--SPLIT--
//-int Solution::uniquePaths(int a, int b) {
//-}

//--SPLIT--
int Solution::uniquePaths(int a, int b) {
	return 0;
}

//--SPLIT--

int main() {
	int testcases;
	cin >> testcases;
    Solution s;
	for (int t = 0; t < testcases; t++) {
		int a, b;
		cin >> a >> b;
		cout << s.uniquePaths(a, b) << "\n";
	}
	return 0;
}
